[ERROR] 25-02-2016 09:49:26.763 [main] model.Usuario:149 - Erro ao Salvar objeto: Usuario{id=null, email=email@example.com, senha=12334, idFacebook=null, nome=Rafael Carlos, sobrenome=Sobrenome, ativo=true, tipousuario=SUPORTE, cartaoCreditoCollection=null, celularRecargaCollection=null, configuracaoCollection=null, recargaGratisCollection=null, recargaAutomaticaCollection=null, feedbackCollection=null, recomendadosCollection=null, carteiraCollection=null, recargaCollection=null}
[ERROR] 25-02-2016 09:49:26.763 [main] model.Usuario:149 - Erro ao Salvar objeto: Usuario{id=null, email=email@example.com, senha=12334, idFacebook=null, nome=Rafael Carlos, sobrenome=Sobrenome, ativo=true, tipousuario=SUPORTE, cartaoCreditoCollection=null, celularRecargaCollection=null, configuracaoCollection=null, recargaGratisCollection=null, recargaAutomaticaCollection=null, feedbackCollection=null, recomendadosCollection=null, carteiraCollection=null, recargaCollection=null}
[ERROR] 25-02-2016 09:49:26.769 [main] model.Usuario:150 - Causa do erro: org.hibernate.exception.GenericJDBCException: could not execute statement
[ERROR] 25-02-2016 09:49:26.769 [main] model.Usuario:150 - Causa do erro: org.hibernate.exception.GenericJDBCException: could not execute statement
